src/schema/v2/artist/meta.ts

```diff
@@ -1,45 +1,91 @@
 import { stripTags, truncate, markdownToText } from "lib/helpers"
 import { compact } from "lodash"
-import { GraphQLString, GraphQLObjectType, GraphQLFieldConfig } from "graphql"
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLFieldConfig,
+  GraphQLEnumType,
+} from "graphql"
 import { ResolverContext } from "types/graphql"
 
 export const metaName = (artist) => {
   if (artist.name) return stripTags(artist.name)
   return "Unnamed Artist"
 }
 
+export const formatDescription = (description: string, blurb?: string) => {
+  return truncate(compact([description, blurb]).join("."), 157)
+}
+
+const ArtistTabEnumType = new GraphQLEnumType({
+  name: "ArtistTab",
```

Comment:
In general, we want to avoid tying our schema to our UI too closely. And I think "tab" gets a little too close to the UI for comfort. "page" is more future proof of a concept. Maybe "mode"? 🤷 

We should also namespace this under any direct types. So this would be `ArtistMetaPage` or `ArtistMetaMode`, in that case.

cc @mzikherman

Reply:
easy peasy -- will change to `Page`